endpoint-service.tf

```diff
@@ -7,5 +7,11 @@ resource "aws_vpc_endpoint_service" "tarball_ingester" {
 
 resource "aws_vpc_endpoint_service_allowed_principal" "tarball_ingester" {
   vpc_endpoint_service_id = aws_vpc_endpoint_service.tarball_ingester.id
```

Comment:
This needs to be gated on something, perhaps `local.tarball_ingester_ucfs_connection`? Because `local.ucfs_account` contains `N/A` for dev & QA envs which is going to result in an invalid ARN.